### PR TITLE
[2.19.x] DDF-UI-115 show inspector thumbnail in table

### PR DESCRIPTION
**Pull request by cassandrabailey293**
#### What does this PR do?
Fixes https://github.com/codice/ddf-ui/issues/115

#### Who is reviewing it? 

@andrewzimmer @hayleynorton @zta6 
#### Select relevant component teams: 

@codice/ui 

#### What are the relevant tickets?
Fixes: https://github.com/codice/ddf-ui/issues/115

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by bdeining** [comment]
build now

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/row.less [hunk 1]
```diff
@@ -39,14 +39,6 @@
   }
 }
 
-@{customElementNamespace}result-row:not(.is-downloadable) {
```
Comment (andrewzimmer) [inline ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/row.less]: This `is-downloadable` is toggled here https://github.com/cassandrabailey293/ddf/blob/DDF-UI-115-thumbnail-display/ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/row.view.js#L113 can it be removed as well?

Comment (cassandrabailey293) [reply ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/row.less]: done 👍

**Comment by andrewzimmer** [comment]
Hero Successful 🎉 
- verified magnifying glass icon appeared and worked correctly

**Comment by bdeining** [comment]
build now

**Pull request merged**